src/pages/about.js

```diff
@@ -1,31 +1,101 @@
 import React, { useState } from 'react'
 import Head from 'next/head'
+import { PageContainer, MainContainer, ContentContainer } from '@/components/Layout'
 import { Circle } from '@/drawings/circle'
 import { Example } from '@/drawings/example'
-import styles from '@/styles/Home.module.css'
 
-export default function Home () {
+export default function Aboute () {
   const [isPaused, setIsPaused] = useState(true)
 
   const togglePause = () => setIsPaused(paused => !paused)
 
   return (
-    <div className={styles.container}>
+    <PageContainer>
       <Head>
         <title>Huygens Pendulum Clock</title>
         <link href='/favicon.ico' rel='icon' />
       </Head>
 
-      <main className={styles.main}>
-        <h2>ABOUT!</h2>
+      <MainContainer>
+        <ContentContainer>
+          <Circle isPaused={isPaused} />
+          <Example />
+        </ContentContainer>
 
-        <button onClick={togglePause}>
-          {isPaused ? 'Unpause' : 'Pause'}
-        </button>
-
-        <Circle isPaused={isPaused} />
-        <Example />
-      </main>
-    </div>
+        <ContentContainer>
+          <h2>ABOUT</h2>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+            {' '}
+            <span
+              onClick={togglePause}
+              onMouseEnter={() => setIsPaused(false)}
+              onMouseLeave={() => setIsPaused(true)}
+              style={{ color: 'teal', fontWeight: 'bold', textDecoration: 'underline' }}
+            >
+              incididunt
+            </span>
```

Comment:
These `<span>`s are disgusting. If this is how we end up doing hoverable text, we'll move this logic into a reusable `/components/_something_` component. I just in-lined it all here for demo purposes.